### DISCUSSION TITLE
[Feature] Support for displaying http content on interact.sh client

### DISCUSSION THREAD

**Discussion by nigamelastic** [projectdiscovery/interactsh#70] [Q&A] [answered]
Right now on sending an http request to the interact.sh url we simply get the following response:

`Recieved HTTP interactionfrom <ip> at <time>`

it would be helpful if we could see the contents of the get/post content of the http request like in `https://webhook.site/`

Ex:
Request at target
`GET customSubdomain.<interactshsubdomin>.interact.sh/someData`

or

```
POST customSubdomain.<interactshsubdomin>.interact.sh
{
value: SomeData
}
```
interactsh-client logs display `SomeData` or the whole json file

**Comment by ehsandeep** [comment, marked as answer]
@nigamelastic we have `-v` flag for the same, and if you prefer the JSON version of it, `-json` flag is also supported.
